datascience/tables.py

```diff
@@ -1250,6 +1292,16 @@ def sample(self, k=None, with_replacement=True, weights=None):
         job  | wage
         a    | 10
         a    | 10
+        >>> jobs.sample(k = 2, weights = make_array(1, 0, 1, 0))
+        Traceback (most recent call last):
+            ...
+        ValueError: probabilities do not sum to 1
+
+        # Weights must be length of table.
+        >>> jobs.sample(k = 2, weights = make_array(1, 0, 0))
+        Traceback (most recent call last):
+            ...
+        ValueError: a and p must have same size
```

Comment:
Could you add a `Raises` section to the docstring to document these `ValueError`s?